remove usage of 'core.addPath'

alfredodeza opened the pull request:
The usage of `core.addPath` was causing the creation of `add-path` into the `dist/index.js` binary via `ncc`. By removing `core.addPath` there are no more warnings. The path is now used directly via the cache instead of just `grype`.

Closes #71

alfredodeza commented:
@jonico I believe this one fixes the problem. Mind double checking the changes? I don't see the warnings anymore

jonico commented:
@alfredodeza: While the code changes look ok to me, you could also just upgrade actions/core to 1.2.6: https://github.com/actions/toolkit/pull/582

alfredodeza commented:
@jonico thanks for the suggestion. The `package.json` file already declares `actions/core` at `^1.2.6`, do you mean some other version? Any pointers would be great, because I do think that bumping that is a good idea as well!

jonico commented:
@alfredodeza: weird, your package.json indicates that should already be at the right version of the package and still [core.add-path uses the old syntax](https://github.com/anchore/scan-action/runs/1344311890?check_suite_focus=true#step:3:38)

Looking into the [source of core.add-path](https://github.com/actions/toolkit/pull/571/files#diff-f63fb32fca85d8e177d6400ce078818a4815b80ac7a3319b60d3507354890992R68)

... could it be that for some reason the env GITHUB_PATH is not set properly in your action?

alfredodeza commented:
@jonico I've updated actions/core *locally* and now the build properly shows the 1.2.6 version. Thanks for catching up. Would love a second look at the changes if you don't mind :) (I dropped the action changes now that 1.2.6 is being used)

jonico commented:
:ship: it!

jonico reviewed (approved):
:shipit:
